Modules/MSCloudLoginAssistant/ConnectionProfile.psm1

```diff
@@ -81,7 +81,7 @@ class Workload
     [string]
     $TenantId
 
-    [string]
+    [securestring]
```

Comment:
Unfortunately this would be a breaking change and would break multiple solutions. We should continue to accept it in as a string and later in code convert it to a secure string

Reply:
Are you sure, that there is any solution, that actually requires the `CertificatePassword` property of the workload class to be a string and works as expected?

I saw that this property is populated by the `Connect-M365Tenant` cmdlet in MSCloudLoginAssistant.psm1 using the `[securestring]$CertificatePassword` parameter. As this parameter is strongly typed to `[securestring]` it will only allow you to bind anything to it, that can be cast to `[securestring]`. 

This means in turn, that `Connect-M365Tenant` will always assign an object of type `[securestring]` to the workload property `CertificatePassword`. The result of this will necessarily be a string that reads: "System.Security.SecureString", as this is the result of a type cast from `[securestring]` to `[string]`.

You can see what I mean in the following minimal example:

```powershell
class workload { [string]$CertificatePassword  }
function foo ([securestring]$CertificatePassword) { 
   $wl = [workload]::new()
   $wl.CertificatePassword = $CertificatePassword
   $wl.CertificatePassword
}
foo -CertificatePassword (ConvertTo-securestring 'foo' -AsPlainText -Force)
```
Judged only by the code in this repository, I don't quite see, how fixing the type of `workload/CertificatePassword` would be a breaking change unless you are referring to solutions, that populate the connection profile outside of the `Connect-M365Tenant` cmdlet.